### PR TITLE
Added config option "suppress-session-update-info"

### PR DESCRIPTION
**Pull request by Dingsel**
Hey, I saw this one feature request (https://github.com/MCXboxBroadcast/Broadcaster/issues/142) where they wanted a features to disable a log message that could potentially get annoying over time. And since I myself find it quite annoying, I added an option into the config with which you can opt in to disable said message.

### CODE REVIEW

File: bootstrap/standalone/src/main/java/com/rtm516/mcxboxbroadcast/bootstrap/standalone/StandaloneMain.java [hunk 1]
```diff
@@ -100,7 +100,9 @@ private static void createSession() throws SessionCreationException, SessionUpda
             try {
                 // Update the session
                 sessionManager.updateSession(sessionInfo);
-                sessionManager.logger().info("Updated session!");
+                if(!config.suppressSessionUpdateInfo()){
```
Comment (rtm516) [inline bootstrap/standalone/src/main/java/com/rtm516/mcxboxbroadcast/bootstrap/standalone/StandaloneMain.java:103]:
```suggestion
                if (!config.suppressSessionUpdateInfo()) {
```

File: bootstrap/standalone/src/main/resources/config.yml [hunk 2]
```diff
@@ -36,3 +36,6 @@ friend-sync:
 
 # Enable debug logs
 debug-log: false
+
+# Suppresses "Updated session!" log that can get spamy over time (feature request https://github.com/MCXboxBroadcast/Broadcaster/issues/142)
```
Comment (rtm516) [inline bootstrap/standalone/src/main/resources/config.yml:40]:
```suggestion
# Suppresses "Updated session!" log into debug
```

File: bootstrap/standalone/src/main/java/com/rtm516/mcxboxbroadcast/bootstrap/standalone/StandaloneMain.java [hunk 3]
```diff
@@ -100,7 +100,9 @@ private static void createSession() throws SessionCreationException, SessionUpda
...
+                    sessionManager.logger().info("Updated session!");
```
Comment (rtm516) [inline bootstrap/standalone/src/main/java/com/rtm516/mcxboxbroadcast/bootstrap/standalone/StandaloneMain.java:104]: Can we make it print this into debug when suppressed, as it's useful for debugging

**Review by rtm516 — CHANGES_REQUESTED** [review]

**Review by rtm516 — APPROVED** [review]

**Comment by Kongtu5i** [comment]
oh my friend, i love you

**Pull request merged**